### PR TITLE
Bucket

### PR DESCRIPTION
**Pull request by kbaran1998**

### CODE REVIEW

File: src/interface_service/requirements.txt [hunk 1]
```diff
@@ -9,4 +9,6 @@ dynaconf==3.1.9
 jinja2==3.1.2
 minio==7.1.8
 google-cloud-pubsub==2.13.0
+joblib==0.14.1
+scikit-learn==0.24.1
```
Comment (kponichtera) [inline src/interface_service/requirements.txt]: Version 0.24.1 is incompatible with Python 3.10 and makes the Docker image building fail.

**Review by kponichtera — CHANGES_REQUESTED** [review]

**Comment by Cornul11** [comment]
You've upgraded the scikit-learn version of the learning_service, not interface_service. The updated version of it also seems to need `gcc` in the container.

**Comment by kbaran1998** [comment]
> You've upgraded the scikit-learn version of the learning_service, not interface_service. The updated version of it also seems to need `gcc` in the container.

I know. I checked if the upgrading version of `scikit-learn` works locally and it does but we are now facing new problems while running in containers: volume permissions.

**Review by kponichtera — APPROVED** [review]
LGTM, but there is still CSV caching to implement.

**Pull request merged**